app/views/renewal/amls_turnover.scala.html

```diff
@@ -20,43 +20,41 @@
 
 @(f: forms.Form2[_], edit: Boolean, businessTypes:Option[List[String]])(implicit lang:Lang,  m: Messages, request: Request[_])
 
-@placeHolder = @{
+@headingText = @{
     businessTypes.map { types =>
-        if (types.length > 1) {
-            "renewal.turnover.title"
-        } else {
+        if (types.length == 1) {
             "renewal.turnover.title.single.service"
+        } else {
+            "renewal.turnover.title"
         }
-    }
+    }.getOrElse("renewal.turnover.title")
```

Comment:
You could refactor this if you like, as you prefer and do something like:
  @headingText = @{
        businessTypes match {
            case Some(types) if types.length == 1 => "renewal.turnover.title.single.service"
            case Some(types) | None => "renewal.turnover.title"
        }
    }

it's more compact and no need to repeat "renewal.turnover.title", but I don't know if it's gonna work :D

Reply:
Good spot